src/modules/projects/components/tables/ProjectExternalSubmissionsTable.tsx

```diff
@@ -38,43 +42,66 @@ const ProjectExternalSubmissionsTable = ({
   projectId: string;
   formDefinitionIdentifier: string;
 }) => {
-  const getColumnDefs = useCallback(() => {
-    return [
-      {
-        header: 'Status',
-        linkTreatment: false,
-        render: (s: ExternalFormSubmissionSummaryFragment) => {
-          const isNew = s.status === ExternalFormSubmissionStatus.New;
-          return (
-            <>
-              <Chip
-                label={capitalize(s.status)}
-                size='small'
-                color={isNew ? 'primary' : 'default'}
-                variant={isNew ? 'filled' : 'outlined'}
-                sx={isNew ? {} : { color: theme.palette.text.secondary }}
-              />
-              {s.spam && (
+  const getColumnDefs = useCallback(
+    (
+      rows: ExternalFormSubmissionSummaryFragment[]
+    ): ColumnDef<ExternalFormSubmissionSummaryFragment>[] => {
+      // Get all unique "summary keys"
+      const summaryKeys = new Set<string>();
+      rows.forEach(({ summaryFields }) =>
+        summaryFields.forEach(({ key }) => summaryKeys.add(key))
+      );
+
+      // Add one column definition for each summary key
+      const defs: ColumnDef<ExternalFormSubmissionSummaryFragment>[] =
+        Array.from(summaryKeys)
+          .sort()
+          .map((key) => ({
+            key,
+            header: key,
+            render: (submission: ExternalFormSubmissionSummaryFragment) =>
+              submission.summaryFields.find((f) => f.key === key)?.value,
+          }));
+
+      return [
+        {
+          header: 'Status',
+          linkTreatment: false,
+          render: (s: ExternalFormSubmissionSummaryFragment) => {
```

Comment:
Maybe avoid single-char vars when referenced on multiple lines? Could replace with destructuring?
```suggestion
          render: ({status, spam, submittedAt}: ExternalFormSubmissionSummaryFragment) => {
```